edge-core/sonic-buildimage
# [Platform][AS9736-64D]: Revise port HW lane list

#### Why I did it
The port HW lane list is incorrect

#### How I did it
Revise the port HW lane list according to the value get from sairedis.rec.

#### How to verify it
Generate default config for hwsku Accton-AS9736-64D and Accton-AS9736-64D-100G, confirm there is no remove port/create port occurred in sairedis.rec

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Ensure to add label/tag for the feature raised. example - PR#2174 under sonic-utilities repo. where, Generic Config and Update feature has been labelled as GCU.

#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
